classes/invitation/invitations/BaseInvitation.php

```diff
@@ -222,6 +223,8 @@ public function setInvitationModel(Invitation $invitationModel)
         $this->keyHash = $invitationModel->keyHash;
         $this->expirationDate = $invitationModel->expiryDate;
         $this->id = $invitationModel->id;
+        $this->email = $invitationModel->email;
```

Comment:
when i try to use and show email in the UI I have to use [getMany()](https://github.com/pkp/pkp-lib/blob/main/classes/invitation/repositories/Invitation.php#L129) function to get data. In that function another function called [constructBOFromModel](https://github.com/pkp/pkp-lib/blob/main/classes/invitation/repositories/Invitation.php#L135). If I removed email and status  from [setInvitationModel](https://github.com/pkp/pkp-lib/blob/main/classes/invitation/invitations/BaseInvitation.php#L220), It will not appear in the UI